addons/survey/views/survey_templates.xml

```diff
@@ -300,6 +300,12 @@
         </div>
     </template>
 
+    <template id="question_answer_image" name="Question answer: image">
+        <div t-if="question.answer_image and label.image" class="d-flex my-2 justify-content-center" style="height: 80%">
+            <div t-if="label.image" t-field="label.image" class="d-flex my-2 align-items-center" t-options='{"widget": "image", "alt-field": "name", "itemprop": "image"}'/>
```

Comment:
Do you really need a div within a div ?
Is there no way to merge those 2 ?

Reply:
done